egs/wsj/s5/steps/nnet3/multilingual/extract_scp.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# This script extract example scp file as egs.*.scp using ranges.*.scp 
+# in multilingual training. 
+
+if [ $# -lt 4 ]; then
+  echo "$0: Usage: $0 num-langs [<egs-dir-lang-1> .. <egs-dir-lang-n>] <ranges.1> <scp-1>"
```

Comment:
this usage message is very unclear, it doesn't explain what the "ranges" or "scp" formats are, and things like `<ranges.1>` are not clear.
In any case, I think there might be a deeper design problem here, making it super slow.